client/src/state-machine.ts

```diff
@@ -43,17 +52,55 @@ type WowState =
     }
   | {
       value: "portfolioFound";
-      context: WowContext & {
-        searchAddrParams: SearchAddressWithoutBbl;
-        searchAddrBbl: string;
-        portfolioData: PortfolioData;
+      context: WowPortfolioFoundContext;
+    }
+  | {
+      value: { portfolioFound: { timeline: "noData" } };
+      context: WowPortfolioFoundContext;
+    }
+  | {
+      value: { portfolioFound: { timeline: "pending" } };
+      context: WowPortfolioFoundContext;
+    }
+  | {
+      value: { portfolioFound: { timeline: "error" } };
+      context: WowPortfolioFoundContext;
+    }
+  | {
+      value: { portfolioFound: { timeline: "success" } };
+      context: WowPortfolioFoundContext & {
+        timelineData: TimelineData;
+      };
+    }
+  | {
+      value: { portfolioFound: { summary: "noData" } };
+      context: WowPortfolioFoundContext;
+    }
+  | {
+      value: { portfolioFound: { summary: "pending" } };
+      context: WowPortfolioFoundContext;
+    }
+  | {
+      value: { portfolioFound: { summary: "error" } };
+      context: WowPortfolioFoundContext;
+    }
```

Comment:
For some reason TypeScript doesn't let me write `summary: "noData"|"pending"|"error"` without introducing bizarre type errors elsewhere in the code, even though the docs actually recommend such a pattern.  Sigh.